### DISCUSSION TITLE
WSQ support

### DISCUSSION THREAD

**Discussion by DevDorrejo** [ImageMagick/ImageMagick#5605] [Development]
I am working with the management of WSQ picture, converting from BMP/JPG/PNG to WSQ.

i will be a big step if imagemagick can support this kind of picture.

**Comment by dlemstra** [comment]
Can you add a link to format specification in case someone wants to add support for this?

**Comment by epcondon** [comment]
I too would welcome WSQ support. It is used in fingerprint image handling. The standard is maintained by the FBI through their Criminal Justice Information Services (CJIS).  website https://fbibiospecs.fbi.gov/biometric-specifications-1-1/document-library. Specification as pdf
[WSQ_Gray-scale_Specification_Version_3_1_Final.pdf](https://github.com/user-attachments/files/26942127/WSQ_Gray-scale_Specification_Version_3_1_Final.pdf)
